tests/components/bring/test_init.py

```diff
@@ -0,0 +1,64 @@
+"""Unit tests for the bring integration."""
+from unittest.mock import Mock
+
+import pytest
+
+from homeassistant.components.bring import (
+    BringAuthException,
+    BringParseException,
+    BringRequestException,
+)
+from homeassistant.components.bring.const import DOMAIN
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+
+from tests.common import MockConfigEntry
+
+
+async def setup_integration(
+    hass: HomeAssistant,
+    bring_config_entry: MockConfigEntry,
+) -> None:
+    """Mock setup of the bring integration."""
+    bring_config_entry.add_to_hass(hass)
+    assert await async_setup_component(hass, DOMAIN, {})
```

Comment:
```suggestion
    assert await hass.config_entries.async_setup(bring_config_entry.entry_id)
```

Reply:
Need to remove the `assert` though, as it may return False for the exception cases @joostlek

Reply:
True!